### ISSUE TITLE
osd full

### ISSUE THREAD

**Issue by 4ss3g4f** [rook/rook#5882]
**Is this a bug report or feature request?**

* Bug Report

**Bug Report**

**Expected behavior:**

**Deviation from expected behavior:**

**How to reproduce it (minimal and precise):**

**File(s) to submit**:

* Cluster CR (custom ressource), typically called `cluster.yaml`, if necessary
* Operator's logs, if necessary
* Crashing pod(s) logs, if necessary

 To get logs, use `kubectl -n <namespace> logs <pod name>`
When pasting logs, always surround them with backticks or use the `insert code` button from the Github UI.
Read [Github documentation if you need help](https://help.github.com/en/articles/creating-and-highlighting-code-blocks).

**Feature Request**

**Are there any similar features already existing:**

**What should the feature do:**

**What would be solved through this feature:**

**Does this have an impact on existing features:**

**Environment**:
* OS (e.g. from /etc/os-release): 
NAME="Flatcar Container Linux by Kinvolk"
ID=flatcar
ID_LIKE=coreos
VERSION=2512.2.1
VERSION_ID=2512.2.1
BUILD_ID=2020-06-16-1044
PRETTY_NAME="Flatcar Container Linux by Kinvolk 2512.2.1 (Oklo)"
ANSI_COLOR="38;5;75"
HOME_URL="https://flatcar-linux.org/"
BUG_REPORT_URL="https://issues.flatcar-linux.org"
FLATCAR_BOARD="amd64-usr"

* Kernel (e.g. `uname -a`):
* Cloud provider or hardware configuration: vmware vsphere
* Rook version (use `rook version` inside of a Rook Pod): v1.3.7
* Storage backend version (e.g. for ceph do `ceph -v`):
* Kubernetes version (use `kubectl version`): v1.16.12
* Kubernetes cluster type (e.g. Tectonic, GKE, OpenShift): baremetal
* Storage backend status (e.g. for Ceph use `ceph health` in the [Rook Ceph toolbox](https://rook.io/docs/rook/master/ceph-toolbox.html)):

![image](https://user-images.githubusercontent.com/22811935/88251263-7a95eb80-ccd4-11ea-903c-477786279ceb.png)

![image](https://user-images.githubusercontent.com/22811935/88251066-d90e9a00-ccd3-11ea-8f37-47817b1794ad.png)

![image](https://user-images.githubusercontent.com/22811935/88251129-0a876580-ccd4-11ea-9fbf-4ec3d95b4661.png)

![image](https://user-images.githubusercontent.com/22811935/88251164-212dbc80-ccd4-11ea-913e-938e78a1b802.png)

**Comment by travisn** [comment]
You could try marking the osd `out` and see if the PGs start moving to other OSDs. But it depends on your replication or erasure coded settings, and CRUSH topology to know if that will work.
```
ceph osd out osd.2
```

**Comment by chintha-v** [comment]
Hi, "ceph df" command shows 94% used. We exec into all pods using rook-ceph-block storageclass pvc and ran "df -kh". Consolidated value is not at all matching with 94%. How to check which one is using how much space from rook-ceph OSDs?

**Comment by chintha-v** [comment]
Hi @travisn , Could you please help us here with above query. Need to identify which pod used more space of rook-ceph block storage.

**Comment by travisn** [comment]
@chintha-v Did you try `ceph osd df`? It should show the usage for each OSD. For more discussion, try the rook slack, or start a [discussion](https://github.com/rook/rook/discussions) instead of commenting on this old issue.

**Comment by chintha-v** [comment]
Hi @travisn ,  I really appreciate you took out time and replied on closed thread. Yeah, all ceph commands are giving stats at OSD level, but we need info which pods are consuming those OSDs and how much they are consuming individually? We tried "df -kh" in individual pods and consolidated rbd values which is lesser than  "ceph osd df" output.
Anyway I created new [https://github.com/rook/rook/discussions/9963](url) on this.
